GeometryService/src/VirtualDetectorMaker.cc

```diff
@@ -572,7 +572,34 @@ namespace mu2e {
               cout << " Constructing " << VirtualDetector::volumeName(VirtualDetectorId::STM_UpStr) << endl;
               cout << "               at local=" << vd->getLocal(VirtualDetectorId::STM_UpStr) << " global="<< vd->getGlobal(VirtualDetectorId::STM_UpStr) <<endl;
            }
-        }
+	}
+
+	//VD 86 Hole filler
+	if ( c.getBool("vd.STMUpStrHole.build", false) ) {
+	  //place this VD 1 cm downstream of the CRS (Cosmic Ray Shield)
+	  // const double y_crv_max       = CRS->getSectorPosition("D").y() + (CRS->getSectorHalfLengths("D"))[1];
+	  // const double yExtentLow      = c.getDouble("yOfFloorSurface.below.mu2eOrigin");
+	  // const double y_vd_center     = (yExtentLow + y_crv_max)/2.0;
+
+	  //const VolumeInfo& parent = _helper->locateVolInfo("STM_CRVShieldPipe");
+	  //bCLHEP::Hep3Vector parentPositionInMu2e = parent.centerInMu2e();
```

Comment:
Unless this is prep for future work, please remove the commented out lines.

Reply:
Done in [8012e85](https://github.com/Mu2e/Offline/pull/1001/commits/8012e85987764b027d1e0f4256fa7640cae3cbbf)

Reply:
THanks.